_episodes/00-setting-the-scene.md

```diff
@@ -17,171 +17,78 @@ you have some basic Python, shell and Git skills."
 ---
 
 ## Introduction
-This lesson teaches intermediate-level software development skills in a way that mimics a typical collaborative 
-software development process. It will teach you to use a number of different software development tools and techniques 
-simultaneously and interchangeably as you would in a real life, rather than teaching them separately. The material is 
-organised into 4 half-day blocks, with the following learning objectives:
+So, you have gained basic software development skills either by self-learning or attending, e.g., a [novice Software 
+Carpentry course][swc-lessons]. You have been applying those skills for a while by writing code to help with your work and you 
+feel comfortable developing code and troubleshooting problems. However, your software 
+has now reached a point where there’s too much code to be kept in one script. It is perhaps involving more 
+researchers (developers) and users, and more collaborative development effort is needed to add new functionality 
+while ensuring previous development efforts remain functional and maintainable. At this point, 
+it becomes particularly important that you rethink the design of your software, including:
 
-1. setting up an own virtual environment using a variety of tools to develop software collaboratively in a team
```

Comment:
With our conversation this morning, do we want to keep these original bullet points as well? Or are we replacing them with the diagram?

Reply:
Maybe ignore this comment - I didn't see the next section yet...